### ISSUE TITLE
Request: Revert change to sandstone

### ISSUE THREAD

**Issue by Sranshaft** [grompe/BTWTweak#11]
Sandstone and its variants should be harvestable with an iron pickaxe or higher similar to how Flowerchild changed loose cobblestone / dirt recently. 

I agree that sand to sandstone should be crafted via hardcore packing; but, it's now become a huge pain in the ass to build anything with sandstone when you get sand piles when breaking anything made from sandstone.

**Comment by grompe** [comment]
Good point, and this sounds more like FC's oversight. I'll look into it.

**Comment by grompe** [comment]
http://sargunster.com/btwforum/viewtopic.php?p=155360#p155360

Looks like it's an intended feature and not an oversight.

**Comment by Sranshaft** [comment]
Jeeze, that's a horrible change...so much for desert base building.

**Comment by grompe** [comment]
As I understand, it is intended to not give the player fire-resistant freely floating blocks at the stage of iron pickaxe.

On related note, I spotted that chisels aren't even set effective on sandstone. Fixed that in BTWTweak v1.1, give it a try.

**Comment by Sranshaft** [comment]
I noticed chiseling smooth and chiselled sandstone drop normal sandstone instead of the intended block.
